apps/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 add_executable(et_contrast src/contrast.cpp)
 target_link_libraries(et_contrast
+	ET::envitools
```

Comment:
All of these should be indented with four spaces. Make sure this is the case for all target_link_libraries commands.

#style